<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>analysis</class>
 <widget class="QWidget" name="analysis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Analysis</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="send_to_server">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>Send files for analysis</string>
          </property>
          <property name="text">
           <string>Send shots</string>
          </property>
          <property name="icon">
           <iconset resource="../../../../usr/local/lib/python2.7/dist-packages/qtutils/icons/icons.qrc">
            <normaloff>:/qtutils/fugue/computer--arrow.png</normaloff>:/qtutils/fugue/computer--arrow.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="server_online">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../../../../usr/local/lib/python2.7/dist-packages/qtutils/icons/icons.qrc">:/qtutils/fugue/status-offline.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="resend_shots_layout">
          <item>
           <widget class="QLabel" name="resend_shots_label">
            <property name="text">
             <string>Files to send: %d</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clear_unsent_shots_button">
            <property name="text">
             <string>Clear</string>
            </property>
            <property name="icon">
             <iconset resource="../../../../usr/local/lib/python2.7/dist-packages/qtutils/icons/icons.qrc">
              <normaloff>:/qtutils/fugue/cross-octagon.png</normaloff>:/qtutils/fugue/cross-octagon.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Server:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="server"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../usr/local/lib/python2.7/dist-packages/qtutils/icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
